internal/manifest/manage.go

```diff
@@ -2,11 +2,18 @@ package manifest
 
 import (
 	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"net/http"
 
 	"github.com/open-feature/cli/internal/filesystem"
+	"github.com/open-feature/cli/internal/flagset"
 	"github.com/spf13/afero"
 )
 
+const flagManifestSchemaURL = "https://raw.githubusercontent.com/open-feature/cli/refs/heads/main/schema/v0/flag-manifest.json"
```

Comment:
should we pick the versioned schema rather than head?

Reply:
That's a good idea. @kriscoleman, we can use Release Please to automagically bump the version before a release. I can show you if you'd like.

Reply:
Sure. Can you please file an issue for this not to forget, and perhaps add a TODO in code?